### PR TITLE
remove lastRun dependancy in start pipeline action

### PR DESCRIPTION
**Pull request by karthikjeeyar**
Refactored the code to remove the unwanted dependancy in start pipeline action

### CODE REVIEW

File: frontend/packages/dev-console/src/utils/pipeline-actions.tsx [hunk 1]
```diff
@@ -29,7 +30,29 @@ const redirectToResourceList = (resource: string) => {
 export const handlePipelineRunSubmit = (pipelineRun: PipelineRun) => {
   redirectToResourceList(`pipelineruns/${pipelineRun.metadata.name}`);
 };
-
+export const createPipelineRunData = (pipeline: Pipeline) => {
```
[the remark this answers is not in the document]
Comment (karthikjeeyar) [reply frontend/packages/dev-console/src/utils/pipeline-actions.tsx]: Yes, now I have modified `newPipelineRun` function to have an options `latestRun` property and removed this `createPipelineRunData` also I have added default serviceAccount for the start pipeline case.

**Comment by invincibleJai** [comment]
I verified locally `start`/ `start last run` works fine.

File: frontend/packages/dev-console/src/utils/pipeline-actions.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (andrewballantyne) [reply frontend/packages/dev-console/src/utils/pipeline-actions.tsx]: Given the names of the other functions in this file - wouldn't we want the `newPipelineRun` to be suffixed by `Data`? (ie `newPipelineRunData` or even `getPipelineData`) so the name indicates it's just supplying data and not actually executing anything?

**Pull request merged**